cmd/controller/main.go

```diff
@@ -147,16 +108,33 @@ func main() {
 		glog.Fatalf("invalid cluster dns specified: %v", err)
 	}
 
-	// set up signals so we handle the first shutdown signal gracefully
-	// TODO: fix stopCh
-	//stopCh := signals.SetupSignalHandler()
+	// TODO: add graceful shutdown, propagate stopCh to run method and to http server
```

Comment:
graceful shutdown is broken - we need to fix it. I think I could add an issue to our backlog.